### PR TITLE
data: prasunkundu.json

### PR DESCRIPTION
**Pull request by PrasunKundu**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5725"></a>

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
mistake

File: data/prasunkundu.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ 
+"name": "Prasun Kundu",
+  "type": "personal",
+  "bio": "Electrical engineering grad who have a deep passion in **Cs and Softwares**. Learning new tech stacks by contributing in OpenSource"
```
Comment (priyankarpal) [inline data/prasunkundu.json:4]: add `,` after the line.
``` suggestion
  "bio": "Electrical engineering grad who has a deep passion in **Cs and Softwares**. Learning new tech stacks by contributing to OpenSource",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/prasunkundu.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "icon": "FaTwitter"
+},
+
+{ "name": "Follow me on Linkedin",
+  "url": "https://www.linkedin.com/in/prasun-kundu-b59009225/",
+  "icon": "FaLinkedin"
+},
+{ "name": "Follow me on Instagram",
+  "url": "https://www.instagram.com/_praasun/",
+  "icon": "FaInstagram",
```
Comment (priyankarpal) [inline data/prasunkundu.json:18]: delete `,` after the line.
``` suggestion
  "icon": "FaInstagram"
}
```

File: data/prasunkundu.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ {"name": "Follow me on Github",
+  "url": "https://github.com/PrasunKundu",
+  "icon": "FaGithub",
```
Comment (priyankarpal) [inline data/prasunkundu.json:21]: also remove `,` from here.
```suggestion
  "icon": "FaGithub"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Also rename to `PrasunKundu.json`

File: data/prasunkundu.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+},
+
+{ "name": "Follow me on Linkedin",
+  "url": "https://www.linkedin.com/in/prasun-kundu-b59009225/",
+  "icon": "FaLinkedin"
+},
+{ "name": "Follow me on Instagram",
+  "url": "https://www.instagram.com/_praasun/",
+  "icon": "FaInstagram"
+}
```
Comment (SaraJaoude) [inline data/prasunkundu.json:19]: missing a comma
```suggestion
},
```

File: data/prasunkundu.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  "icon": "FaTwitter"
+},
+
+{ "name": "Follow me on Linkedin",
+  "url": "https://www.linkedin.com/in/prasun-kundu-b59009225/",
+  "icon": "FaLinkedin"
+},
+{ "name": "Follow me on Instagram",
+  "url": "https://www.instagram.com/_praasun/",
+  "icon": "FaInstagram"
```
Comment (PrasunKundu) [inline data/prasunkundu.json:18]: ok

File: data/prasunkundu.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+}
+ {"name": "Follow me on Github",
+  "url": "https://github.com/PrasunKundu",
+  "icon": "FaGithub"
+}
+]
```
Comment (PrasunKundu) [inline data/prasunkundu.json:24]: ,

**Comment by PrasunKundu** [comment]
hey eddie i am waiting for your approval. :)

**Pull request closed without merging**